### PR TITLE
Update README

### PR DESCRIPTION
**Pull request by henrylamchan**
# Overview
[Asana](https://app.asana.com/0/1165454451722972/1166016335669035/f)

- Minor changes to README referencing WorkOS.js embed instead of IdP Link
- Added link to WorkOS.js embed docs in the README

### CODE REVIEW

File: README.md [hunk 1]
```diff
@@ -1,66 +1,55 @@
...
-```html
-<div 
+```
+<div
   class='workos-container'
-  data-prop-domain='CURRENT-USER-DOMAIN'
-  data-prop-name='CURRENT-USER-COMPANY-NAME'
-  data-prop-project-id='WORKOS_PROJECT_ID'
-  data-prop-publishable-key='WORKOS_PUBLISHABLE_KEY'
+  data-prop-domain='{foo-corp.com}'
```
Comment (sbauch) [inline README.md]: we cant use code injection here, not sure what's best though...

File: README.md [hunk 2]
```diff
@@ -1,66 +1,55 @@
...
 
-### Installation
+To use WorkOS.js, you'll need to know your `Project ID`, your `Publishable Key`. You'll also need to supply the specific domain of an enterprise customer who is creating the new Identity Provider connection. Let's get started.
 
-WorkOS IdP Link is available as a Javascript embed via the WorkOS CDN. WorkOS is always adding new Identity Providers and keeping the embed updated, so it's important you include the script from the WorkOS CDN as opposed to copying it into your codebase. Include the following script tag on your website:
+## Set up the Element
 
-```html
-<script async type="text/javascript" src="https://cdn.workos.com/idp-link.min.js"</script>
+Add the script to your application's website.
```
Comment (sbauch) [inline README.md]: can we add something like "and include on any page where you allow your user to set up and configure SSO connections." Then something about async or defer, and where to put on page etc. This is a nice article you can borrow from - https://developers.google.com/web/fundamentals/performance/optimizing-content-efficiency/loading-third-party-javascript

Comment (henrylamchan) [reply README.md]: Sure

File: README.md (hunk 1 above)
Comment (henrylamchan) [reply README.md]: could leave as is or copy over the values used in index.erb even though they're not actual values as well

**Comment by sbauch** [comment]
added @grinich sounds like from the discussion on card he's the right person to approve

**Comment by grinich** [comment]
@henrylamchan I would prefer to have our docs duplicated here. Instead, let's make the relevant changes (if any) directly in the `docs` package, and this Readme can be specifically only about the demo code.

**Review by grinich — DISMISSED** [review]
@henrylamchan I would prefer to have our docs duplicated here. Instead, let's make the relevant changes (if any) directly in the docs package, and this Readme can be specifically only about the demo code.

File: README.md [hunk 3]
```diff
@@ -31,34 +20,35 @@ Then add an element to your DOM with the class `workos-container`
 ```html
 <div 
   class='workos-container'
-  data-prop-domain='CURRENT-USER-DOMAIN'
-  data-prop-name='CURRENT-USER-COMPANY-NAME'
   data-prop-project-id='WORKOS_PROJECT_ID'
   data-prop-publishable-key='WORKOS_PUBLISHABLE_KEY'
+  data-prop-domain='CURRENT_USER_DOMAIN'
+  data-prop-name='CURRENT_USER–_OMPANY_NAME'
```
Comment (sbauch) [inline README.md]: 🤔

File: README.md [hunk 4]
```diff
@@ -1,28 +1,17 @@
-# IdP Link
+# workosjs-demo
 
-WorkOS offers a Javascript based embeddable widget that allows your Enterprise users to configure their Single Sign-On Identity Provider themselves. Using IdP Link allows you to reduce the back and forth between your team and your new Enterprise customer, getting the Enterprise onboarded quicker and using your application.
+An example Sinatra app demonstrating how onboarding an Enterprise may work utilizing the WorkOS.js embed.
 
-![alt text](./IdPLink.gif)
+![alt text](./WorkOSjs.gif)
```
Comment (sbauch) [inline README.md]: "alt text" isnt great alt text lol, mind replacing?

File: README.md (hunk 3 above)
Comment (henrylamchan) [reply README.md]: 🤔

Comment (sbauch) [reply README.md]:
```suggestion
  data-prop-name='CURRENT_USER_COMPANY_NAME'
```

**Review by sbauch — APPROVED** [review]
lgtm!

**Comment by henrylamchan** [comment]
Looks like im blocked from merging till the 'changes requested' review has been resolved... @grinich

**Comment by henrylamchan** [comment]
oops, wrong repo =) no bors =)

**Pull request merged**